workitem.go

```diff
@@ -185,7 +185,23 @@ func (c *WorkitemController) Delete(ctx *app.DeleteWorkitemContext) error {
 	return application.Transactional(c.db, func(appl application.Application) error {
 		err := appl.WorkItems().Delete(ctx, ctx.ID)
 		if err != nil {
-			return jsonapi.JSONErrorResponse(ctx, errs.Wrap(err, fmt.Sprintf("Error deleting work item")))
+			return jsonapi.JSONErrorResponse(ctx, errs.Wrapf(err, "error deleting work item %s", ctx.ID))
+		}
+		// Now find and delete all work item links where the current deleted
+		// work item is a part of.
+		wilList, err := appl.WorkItemLinks().ListByWorkItemID(ctx, ctx.ID)
```

Comment:
Done in 	2ed9d6f. Thanks for the suggestion as it makes the code much faster and easier to use, read, and maintain.